### ISSUE TITLE
Portal and Item Emotes, Cooldown timers, and Use Level Requirements

### ISSUE THREAD

**Issue by kdmxp256** [ACEmulator/ACE#1243]
### Bug Report or Feature Request (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [X] feature request
```
The following feature requests are in regards to the content update #21 that is pending in 16PY-Patches repo:

1) Portals and items need to be able to process emotes: Training Academy exit portals use emotes to reset a player's Sanctuary position as they are level the TA; Protection Orb send a tell to the player when it is picked up
2) New data fields have been added, per PCAP and Lifestoned data, that need coding to handle, such as cooldowns and level activation requirement: Facility Hub Portal Gem has both a cooldown timer and a use level requirement and the Blackmoor's Favor gem has a cooldown.

Both of these points are backed by data from PCAPs and the Weenie data posted on the Lifestoned site.

**Comment by LtRipley36706** [comment]
Resolved the Portal emote issue with #1238

**Comment by kdmxp256** [comment]
Confirmed all issues in this ticket resolved
